File: C++/remove_element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int tmp = 0;
        int len = nums.size();
        
        for(int i = 0; i < len; i++){
            if(nums[i-tmp] == val){
                nums.erase(nums.begin() + (i-tmp));
                tmp++;
            }
        }
        return nums.size();
    }
};

File: Python3/remove_element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        tmp = 0
        
        for i in range(len(nums)):
            if nums[i-tmp] == val:
                nums.pop(i-tmp)
                tmp += 1
                
        return len(nums)
